.travis.yml

```diff
@@ -22,8 +22,61 @@ install:
   - pip3 install -r $TRAVIS_BUILD_DIR/ui_testing/requirements.txt
 
 script:
-  - export TESTS='test09_my_profile.py:'
+  - export TESTS='test06_orders.py:'
   - if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then export TESTS=''; fi
   - cd $TRAVIS_BUILD_DIR && export PYTHONPATH='./'
-  - xvfb-run -a nosetests-3.4 -vs --logging-level=WARNING ui_testing/testcases/basic_tests/test02_articles.py:ArticlesTestCases.test019_download_article_sheet --tc-file=config.ini
+
+
+
+  
+  # - xvfb-run -a nosetests-3.4 -vs --nologcapture ui_testing/testcases/header_tests/$TESTS --tc-file=config.ini
+  - xvfb-run -a nosetests -vs --nologcapture --tc-file=config.ini ui_testing/testcases/basic_tests/test06_orders.py:OrdersTestCases.test012_duplicate_many_orders --tc-file=config.ini
```

Comment:
This PR changed lots of testcases, so please run all of them

Reply:
@0xIslamTaha we didn't have orders test cases yet so what do you mean by it changed a lots of test cases

Reply:
@dianaeldemerdash not true, check the `File Changed` tab please